### PR TITLE
Make logo selector #brand-logo and reduce specificity

### PR DESCRIPTION
**Pull request by sg00dwin**
fixes https://github.com/openshift/origin/issues/4617

### CODE REVIEW

**Comment by sg00dwin** [comment]
@jwforres

File: assets/app/styles/_core.less [hunk 1]
```diff
@@ -76,6 +66,12 @@ html, body {
     margin-top: 22px;
   }
 }
+#brand-logo {
+  background: url('@{image-path}/logo-origin-thin.svg');
```
Comment (spadgett) [inline assets/app/styles/_core.less]: Use `background-image` instead of the shorthand style since we're only setting one property.

**Comment by liggitt** [comment]
this is probably our last chance to pick an id... we happy with `#brand-logo` or do we want something like `#header-logo`, etc?

File: assets/app/styles/_core.less (hunk 1 above)
Comment (spadgett) [inline assets/app/styles/_core.less]: Or add `no-repeat` to the background shorthand property.

**Comment by spadgett** [comment]
I think `header-logo` or `banner-logo` is better.

**Comment by liggitt** [comment]
@spadgett either in this one or separately, we might want to change the id of `openshift-logo` to make it more generic and clarify it is the link, not the logo itself

**Comment by sg00dwin** [comment]
pr updated

**Comment by liggitt** [comment]
lgtm, [merge]

**Pull request merged**